pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -125,10 +126,14 @@ func (pg *pgcontroller) createNormalPodPGIfNotExist(pod *v1.Pod) error {
 	return pg.updatePodAnnotations(pod, pgName)
 }
 
+// newPGOwnerReferences create ownerreference for pg,the pod that create by other controller use pod as the ownerreference
 func newPGOwnerReferences(pod *v1.Pod) []metav1.OwnerReference {
 	if len(pod.OwnerReferences) != 0 {
 		for _, ownerReference := range pod.OwnerReferences {
-			if ownerReference.Controller != nil && *ownerReference.Controller {
+			if ownerReference.Controller != nil &&
+				*ownerReference.Controller &&
+				ownerReference.APIVersion == batchv1alpha1.SchemeGroupVersion.String() &&
+				ownerReference.Kind == "Job" {
```

Comment:
hm... how this PR fix the problem?

Reply:
Only pods created by the controller of volcano use the owner of the pod as the owner of the pg,in other case,pg will use pod as the owner of pg.